dts/arm/nordic/nrf9160.dtsi

```diff
@@ -106,6 +108,21 @@
 			reg = <0xff8000 0x1000>;
 			status = "okay";
 		};
+
+		cryptocell: crypto@50840000 {
```

Comment:
Can you move it before gpiote, so GPIOTE, SPU and CRYPTOCELL Are ordered alphabetically?
(FICR and UICR are not peripherals, stictly speaking)

Reply:
Done.